tests/run-pass/move_fn_closure.rs

```diff
@@ -0,0 +1,24 @@
+struct Foo<'a>(&'a mut bool);
```

Comment:
You should probably merge this file with the existing https://github.com/solson/miri/blob/master/tests/run-pass/closure-drop.rs.